docs/API/SandboxSetup.md

```diff
@@ -4,48 +4,52 @@ Chaincode developers need a way to test and debug their chaincode without having
 
 Multiple terminal windows inside the Vagrant development environment are required. One Vagrant terminal runs the validating peer; another Vagrant terminal runs the chaincode; the third Vagrant terminal runs the CLI or REST API commands to execute transactions. When running with security enabled, an additional fourth Vagrant terminal window is required to run the <b>Certificate Authority (CA)</b> server. Detailed instructions are provided in the sections below:
 
-* [Security Setup (optional)](#security-setup-optional)
+* [[Optional] Security Setup](#optional-security-setup)
 * [Vagrant Terminal 1 (validating peer)](#vagrant-terminal-1-validating-peer)
 * [Vagrant Terminal 2 (chaincode)](#vagrant-terminal-2-chaincode)
 * [Vagrant Terminal 3 (CLI or REST API)](#vagrant-terminal-3-cli-or-rest-api)
     * [Chaincode deploy via CLI and REST](#chaincode-deploy-via-cli-and-rest)
     * [Chaincode invoke via CLI and REST](#chaincode-invoke-via-cli-and-rest)
     * [Chaincode query via CLI and REST](#chaincode-query-via-cli-and-rest)
-* [Removing temporary files when security is enabled](#removing-temporary-files-when-security-is-enabled)
+* [[Optional] Removing temporary files when security is enabled](#optional-removing-temporary-files-when-security-is-enabled)
 
 See the [logging control](../dev-setup/logging-control.md) reference for information on controlling
 logging output from the `peer` and chaincodes.
 
-###Security Setup (optional)
+## [Optional] Security Setup
 From your command line terminal, move to the `devenv` subdirectory of your workspace environment. Log into a Vagrant terminal by executing the following command:
 
     vagrant ssh
 
 To set up the local development environment with security enabled, you must first build and run the <b>Certificate Authority (CA)</b> server:
 
     cd $GOPATH/src/github.com/hyperledger/fabric
-    make membersrvc && membersrvc
+    make membersrvc
+    cd build/bin
```

Comment:
Something likely failed during the `vagrant up` process. The `build/bin` folder should have been put on your PATH. See https://github.com/hyperledger/fabric/blob/master/devenv/setup.sh#L110